prowler/providers/aws/lib/policy_condition_parser/policy_condition_parser.py

```diff
@@ -1,5 +1,8 @@
 def is_condition_block_restrictive(
-    condition_statement: dict, source_account: str, is_cross_account_allowed=False
+    condition_statement: dict,
+    source_account: str,
+    is_cross_account_allowed=False,
+    org_id: str = None,
```

Comment:
Do we need this? Can we not just check if the condition is set?

Reply:
added org_id since the format for this sort of conditions where like: `"aws:resourceaccount": "<aws_account>"` but for the new condition the org_id it´s needed. This way we ensure that the condition it´s well applied.

Reply:
I'm more in favor of just checking if this condition is met. This function just checks if the `condition` block of an IAM policy is restrictive, which means that at least one of the conditions down there is set and with an specific value, not an `*`.

Also, I don't like the need of changing the function result since applies to many other checks.